cloud_funcs/solvers/text_analyzer.py

```diff
@@ -7,19 +8,22 @@
 CLAUDE_API_KEY = os.environ.get('CLAUDE_API_KEY')
 chat = None
 
-def summarize(context):
+def summarize(context, lang = "en"):
     print("context", context)
     global chat
 
+    messagesFile = open(f"./i18n/${lang}.json")
```

Comment:
please close file at the end, since python functions can be run on same runtime by multiple calls

Reply:
or use pattern with...open